test/lib/utils/addresses.ts

```diff
@@ -220,6 +220,18 @@ export const TOKENS: Record<number, Record<string, TestToken>> = {
             slot: 51,
         },
     },
+    [ChainId.PLASMA]: {
+        USDT: {
+            address: '0xB8CE59FC3717ada4C02eaDF9682A9e934F625ebb',
+            decimals: 6,
+            slot: 51,
+        },
+        weETH: {
+            address: '0xA3D68b74bF0528fdD07263c60d6488749044914b',
+            decimals: 18,
+            slot: 3, // not true
```

Comment:
Slot is actually not required. Feel free to remove the attribute if it's not relevant to your current tests.